src/readpst.c

```diff
@@ -425,7 +425,7 @@ int process(pst_item *outeritem, pst_desc_tree *d_ptr)
         }
         pst_freeItem(item);
     }
-    success &= ff.found_count == ff.stored_count;
+    success &= (! outeritem->folder || ff.found_count == ff.stored_count) && ff.skip_count == 0;
```

Comment:
In the ~10 psts I've tested there are still often other unknown types - "IPM.AbchPerson", "IPM.DistList", "IPM.Microsoft....", "IPM.MS....".  My plan right now is to dump all unknown nodes as text files with their MAPI properties (alternatively we could dump something like a MSG but that seems more complicated).